samples/src/Microsoft.Azure.IIoT.App/src/Models/DiscovererInfoRequested.cs

```diff
@@ -48,11 +48,15 @@ public class DiscovererInfoRequested
         /// List of preset discovery urls to use
         /// </summary>
         public List<string> RequestedDiscoveryUrls { get; set; }
+
         /// <summary>
         /// Add url
         /// </summary>
         public void AddDiscoveryUrl(string url)
         {
```

Comment:
Sorry, I didn't see this before. .cs files should actually be formatted with help of the .editorconfig file. Razor files manually.